Tasks/Common/webdeployment-common/msdeployutility.ts

```diff
@@ -27,30 +26,33 @@ var parseString = require('xml2js').parseString;
 export function getMSDeployCmdArgs(webAppPackage: string, webAppName: string, publishingProfile,
                              removeAdditionalFilesFlag: boolean, excludeFilesFromAppDataFlag: boolean, takeAppOfflineFlag: boolean,
                              virtualApplication: string, setParametersFile: string, additionalArguments: string, isParamFilePresentInPacakge: boolean,
-                             isFolderBasedDeployment: boolean, useWebDeploy: boolean) : string {
+                             isFolderBasedDeployment: boolean, useWebDeploy: boolean, overRideParams : string[]) : string {
 
     var msDeployCmdArgs: string = " -verb:sync";
 
     var webApplicationDeploymentPath = (virtualApplication) ? webAppName + "/" + virtualApplication : webAppName;
     
     if(isFolderBasedDeployment) {
         msDeployCmdArgs += " -source:IisApp=\"" + webAppPackage + "\"";
-        msDeployCmdArgs += " -dest:iisApp=\"" + webApplicationDeploymentPath + "\",";
+        msDeployCmdArgs += " -dest:iisApp=\"" + webApplicationDeploymentPath + "\"";
     }
     else {       
         msDeployCmdArgs += " -source:package=\"" + webAppPackage + "\"";
 
         if(isParamFilePresentInPacakge) {
-            msDeployCmdArgs += " -dest:auto,";
+            msDeployCmdArgs += " -dest:auto";
```

Comment:
I believe we closed that we will not pass the destination provider as iisApp as it was breaking in few cases.